packages/react/src/components/DatePicker/DatePicker.js

```diff
@@ -405,6 +393,19 @@ export default class DatePicker extends Component {
         this.cal.set('maxDate', maxDate);
       }
     }
+
+    // Coordinate when the given `value` prop changes. When this happens, we
+    // should update the calendar to the new value.
+    if (prevValue !== value) {
+      if (this.cal) {
+        this.cal.setDate(this.props.value);
+        this.updateClassNames(this.cal);
+      } else {
+        if (this.inputField) {
```

Comment:
```suggestion
      } else if (this.inputField) {
```

just wondering is there any reason for splitting it up like this?

Reply:
No reason! Just copying over from previous source. Great catch, let me add this in.